### ISSUE TITLE
Java Intellisense/autocomplete not working in .ipynb

### ISSUE THREAD

**Issue by samax-dx** [redhat-developer/vscode-java#2568]
Intellisense won't work **at all** when using java kernel in Jupyter notebook file. Tried - BeakerX, iJava, and Ganymede; no luck.  

##### Environment
- Operating System: Windows 10 Pro x64
- JDK version: 17.0.1
- Visual Studio Code version: 1.69.1
- Java extension version: v1.8.0

##### Steps To Reproduce
1. Open .ipynb file
2. Choose Java kernel
3. Try writing any Java code

##### Logs
[vscode-log.txt](https://github.com/redhat-developer/vscode-java/files/9118251/vscode-log.txt)
[java-language-server-log.txt](https://github.com/redhat-developer/vscode-java/files/9118250/java-language-server-log.txt)

##### Current Result
No Intellisense/autocompletion

##### Expected Result
Intellisense/autocompletion

**Comment by fbricon** [comment]
Indeed, Jupyter notebooks are not supported at all. Noone ever asked for it. I never used jupyter notebooks, much less java-based ones. But my guess is providing support will most likely require non-trivial investment. Do you feel like contributing it?

**Comment by samax-dx** [comment]
> Indeed, Jupyter notebooks are not supported at all. Noone ever asked for it. I never used jupyter notebooks, much less java-based ones. But my guess is providing support will most likely require non-trivial investment. Do you feel like contributing it?

I never developed vscode extensions and I suspect this is an advanced task in the field. Sadly so, can't contribute right now.
